packages/cache-core/src/strategies/fileSystem.ts

```diff
@@ -1,21 +1,30 @@
 import fs from 'node:fs/promises'
+import { existsSync } from 'node:fs'
 import path from 'node:path'
 import type { CacheStrategy, CacheEntry, CacheContext } from '@dbbs/next-cache-handler-common'
 
 export class FileSystemCache implements CacheStrategy {
   async get(pageKey: string, cacheKey: string, ctx: CacheContext): Promise<CacheEntry | null> {
-    const data = await fs.readFile(path.join(ctx.serverCacheDirPath, pageKey, `${cacheKey}.json`), {
-      encoding: 'utf-8'
-    })
+    const pathToCacheFile = path.join(ctx.serverCacheDirPath, pageKey, `${cacheKey}.json`)
+
+    if (!existsSync(pathToCacheFile)) {
+      throw `no cache file ${pathToCacheFile} yet`
```

Comment:
if there is no file, we should return `null`, + we have logging and try/catch directly in cache handler to manage such issues

Reply:
so we should check file availability here, probably we can just improve loggin inside cacheHandler for that

Reply:
fixed